manual/src/cmds/intf-c.etex

```diff
@@ -141,24 +141,22 @@ arguments and returning a native C value. The second function,
 often called the ``stub code'', is a simple wrapper around the first
 function that converts its arguments from OCaml values to C values,
 call the first function, and convert the returned C value to OCaml
```

Comment:
It's not strictly part of the change, but while checking the context of the line below:
```suggestion
calls the first function, and converts the returned C value to an OCaml
```